rest-new/src/main/java/org/jboss/pnc/rest/RequestLoggingFilter.java

```diff
@@ -75,7 +75,12 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
             MDCUtils.addProcessContext(logProcessContext);
         }
 
-        User user = userService.currentUser();
+        User user = null;
+        try {
+            user = userService.currentUser();
+        } catch (IllegalStateException e) {
+            //user not found.. proceed
+        }
```

Comment:
I would maybe move the bellow if inside the try?

Reply:
sure